tests/unit/Trackable.js

```diff
@@ -201,9 +202,10 @@ define([
 
 				var observations = [];
 				// temporarily hide this so we don't load everything
+				var fetchSync = bigFiltered.fetchSync;
 				bigFiltered.fetchSync = null;
 				var bigObserved = bigFiltered.track();
-				delete bigFiltered.fetchSync;
+				bigFiltered.fetchSync = fetchSync;
```

Comment:
It was a workaround necessitated by the fact that the prototype wasn't setup properly, this has been resolved by using `Object.create` in `_createSubCollection`, so I undid this change.